cmake/modules/llext.cmake

```diff
@@ -0,0 +1,111 @@
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
Not related to this specific line, but comment relating to https://github.com/zephyrproject-rtos/zephyr/pull/67431/commits/ee9a514531497917b596a65a97d14865d0e1acf9 + https://github.com/zephyrproject-rtos/zephyr/pull/67431/commits/113902e85e9dbb0e35e658906fdefb8a4a4deed3 vs https://github.com/zephyrproject-rtos/zephyr/pull/67431/commits/d435f7b0c0e31d83e445db3dc5e5a09250240f8f: 
reviewing PR's where code is moved around between commits makes it very hard to review a PR commit-by-commit.